sklearn/externals/joblib/__init__.py

```diff
@@ -1,133 +1,3 @@
-"""Joblib is a set of tools to provide **lightweight pipelining in
-Python**. In particular:
+# Import necessary to preserve backward compatibliity of pickles
```

Comment:
As far as I understand, this is necessary to allow loading pickles created with the vendored joblib. Currently there is some number of those, but after it is merged and a few releases their relative amount should decrease. So I would say maybe we can deprecate it in 0.22 or 0.23.

Though another concern is to make sure people are not using this module as an alias for joblib. So maybe we should raise a warning here.

Will add a comment or warning once we reach an agreement about how to handle this.

Reply:
I may be missing the bigger picture, but I'd be in favor of raising a deprecation warning here since this is our only way to discourage users from using `sklearn.externals.joblib`.